app-backup/deja-dup/deja-dup-42.7.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson eutils gnome2-utils vala xdg
+
+DESCRIPTION="Simple backup tool using duplicity back-end"
+HOMEPAGE="https://gitlab.gnome.org/World/deja-dup"
+SRC_URI="https://gitlab.gnome.org/World/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	>=app-crypt/libsecret-0.18.6[vala]
+	>=dev-libs/glib-2.64:2[dbus]
+	>=dev-libs/json-glib-1.2
+	dev-libs/libgpg-error
+	>=gui-libs/libhandy-1.0
+	>=net-libs/libsoup-2.48
+	>=x11-libs/gtk+-3.22:3
+	>=x11-libs/libnotify-0.7
+	>=app-backup/duplicity-0.7.14
+"
+
+RDEPEND="${COMMON_DEPEND}
+	gnome-base/dconf
+	gnome-base/gvfs[fuse]
+"
+DEPEND="${COMMON_DEPEND}
+	$(vala_depend)
+	virtual/pkgconfig
+	dev-util/intltool
+	dev-util/itstool
+	sys-devel/gettext
```

Comment:
Sort (place virtual/pkgconfig below everything in this selected block)

Reply:
done.